pkgs/development/python-modules/pydeako/default.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pythonOlder,
+  setuptools,
+  zeroconf,
+}:
+
+buildPythonPackage rec {
+  pname = "pydeako";
+  version = "0.5.4";
+  pyproject = true;
+
+  disabled = pythonOlder "3.11";
+
+  src = fetchFromGitHub {
+    owner = "DeakoLights";
+    repo = "pydeako";
+    rev = "refs/tags/${version}";
+    hash = "sha256-Z0H5VhWfjmvvCGTX//hds9dwk2wJSPXckNac1PkQZNA=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ zeroconf ];
+
+  # Module has no tests
+  doCheck = false;
```

Comment:
https://github.com/DeakoLights/pydeako/blob/main/pydeako/deako/test_deako.py

has quite a number of tests